cmd/xdpcap/flags.go

```diff
@@ -0,0 +1,110 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/pkg/errors"
+)
+
+// nopWriter implements io.Writer and discards everything written to it
+type nopWriter struct {
+}
+
+func (nopWriter) Write(p []byte) (int, error) {
+	return len(p), nil
+}
+
+type flags struct {
+	*flag.FlagSet
```

Comment:
Split this into `config struct{}` and a free function that uses a FlagSet instead? I'm not a fan of mixing the source of config with the config itself.

Reply:
Merged the `captureOpts` and `flags` types, and `newFlags()` and `flags.Parse` into `parseFlags()`, which I think is clearer.